service/src/main/java/io/camunda/service/AuthorizationServices.java

```diff
@@ -46,6 +50,29 @@ public SearchQueryResult<AuthorizationEntity> search(final AuthorizationQuery qu
     return authorizationSearchClient.searchAuthorizations(query, authentication);
   }
 
+  public Set<String> fetchAssignedPermissions(
+      final String ownerType,
```

Comment:
🔧 NABD but I believe as `ownerId` is an entirely unique field we can make this method simpler by removing the `ownerType`. I think this works in all instances, for example if I am using basic-auth then the authentication object contains my user key which is the value stored on the authorization as the owner.